packages/labs/analyzer/src/test/lit-element/properties_test.ts

```diff
@@ -268,5 +269,26 @@ for (const lang of languages) {
     }
   });
 
+  test('property with an unsupported name type', ({analyzer, element}) => {
+    // This currently results in two diagnostics: one for the
+    // `LitElement`-specific part of the analysis and one for the vanilla class
+    // analysis.
```

Comment:
I was wondering about this... can we make one take precedence and squash the other?